<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImportKey</class>
 <widget class="QDialog" name="ImportKey">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>530</width>
    <height>389</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="10" column="2" colspan="3">
      <widget class="QLineEdit" name="passwordHash">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="8" column="3" colspan="2">
      <widget class="QCheckBox" name="showPassword">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Show</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0" colspan="2">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>&amp;Address:</string>
       </property>
       <property name="buddy">
        <cstring>address</cstring>
       </property>
      </widget>
     </item>
     <item row="3" column="4">
      <widget class="QToolButton" name="openKey">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item row="8" column="0" colspan="2">
      <widget class="QRadioButton" name="newPassword">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>New &amp;Passphrase:</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup</string>
       </attribute>
      </widget>
     </item>
     <item row="4" column="2" colspan="3">
      <widget class="QLineEdit" name="address">
       <property name="placeholderText">
        <string>Place the address of the key here</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0" colspan="2">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>&amp;Name:</string>
       </property>
       <property name="buddy">
        <cstring>name</cstring>
       </property>
      </widget>
     </item>
     <item row="6" column="0" colspan="2">
      <widget class="QRadioButton" name="noPassword">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>&amp;Master Passphrase</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup</string>
       </attribute>
      </widget>
     </item>
     <item row="11" column="2" colspan="3">
      <widget class="QLineEdit" name="hint"/>
     </item>
     <item row="7" column="0" colspan="2">
      <widget class="QRadioButton" name="oldPassword">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>&amp;Old Passphrase</string>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup</string>
       </attribute>
      </widget>
     </item>
     <item row="8" column="2">
      <widget class="QLineEdit" name="password">
       <property name="placeholderText">
        <string>Enter the passphrase you wish to use for the key here</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2" colspan="3">
      <widget class="QLineEdit" name="name">
       <property name="placeholderText">
        <string>Enter this key's name here</string>
       </property>
      </widget>
     </item>
     <item row="10" column="0" colspan="2">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Passphrase &amp;Hash:</string>
       </property>
       <property name="buddy">
        <cstring>passwordHash</cstring>
       </property>
      </widget>
     </item>
     <item row="11" column="0" colspan="2">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Passphrase Hin&amp;t:</string>
       </property>
       <property name="buddy">
        <cstring>hint</cstring>
       </property>
      </widget>
     </item>
     <item row="6" column="2" colspan="3">
      <widget class="QLabel" name="masterLabel">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Use same passphrase for the key as for the master.</string>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="2" colspan="2">
      <widget class="QLineEdit" name="key">
       <property name="placeholderText">
        <string>Brain wallet (no key file)</string>
       </property>
       <property name="clearButtonEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="0" colspan="2">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>&amp;Key:</string>
       </property>
       <property name="buddy">
        <cstring>key</cstring>
       </property>
      </widget>
     </item>
     <item row="7" column="2" colspan="3">
      <widget class="QLabel" name="oldLabel">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Use the same passphrase as in the key file.</string>
       </property>
      </widget>
     </item>
     <item row="5" column="2" colspan="3">
      <widget class="QLineEdit" name="addressOut">
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="placeholderText">
        <string>Unknown Address</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>5</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="action">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="cancel">
       <property name="text">
        <string>Cancel</string>
       </property>
       <property name="shortcut">
        <string>Esc</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="import_2">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>&amp;Import</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>name</tabstop>
  <tabstop>key</tabstop>
  <tabstop>openKey</tabstop>
  <tabstop>address</tabstop>
  <tabstop>addressOut</tabstop>
  <tabstop>noPassword</tabstop>
  <tabstop>oldPassword</tabstop>
  <tabstop>newPassword</tabstop>
  <tabstop>password</tabstop>
  <tabstop>showPassword</tabstop>
  <tabstop>passwordHash</tabstop>
  <tabstop>hint</tabstop>
  <tabstop>cancel</tabstop>
  <tabstop>import_2</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>import_2</sender>
   <signal>clicked()</signal>
   <receiver>ImportKey</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>519</x>
     <y>378</y>
    </hint>
    <hint type="destinationlabel">
     <x>449</x>
     <y>388</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancel</sender>
   <signal>clicked()</signal>
   <receiver>ImportKey</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>433</x>
     <y>378</y>
    </hint>
    <hint type="destinationlabel">
     <x>351</x>
     <y>388</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>newPassword</sender>
   <signal>pressed()</signal>
   <receiver>password</receiver>
   <slot>setFocus()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>80</x>
     <y>211</y>
    </hint>
    <hint type="destinationlabel">
     <x>185</x>
     <y>215</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
